line-bot-api-client/src/test/java/com/linecorp/bot/client/LineMessagingClientImplIntegrationTest.java

```diff
@@ -50,20 +83,46 @@
     public static final URL TEST_RESOURCE = ClassLoader.getSystemResource("integration_test_settings.yml");
     private LineMessagingClient target;
     private String userId;
+    private IntegrationTestSettings settings;
 
     @Before
     public void setUp() throws IOException {
+        // Do not run all test cases in this class when src/test/resources/integration_test_settings.yml doesn't
+        // exist.
         Assume.assumeTrue(TEST_RESOURCE != null);
```

Comment:
I'm sorry but could you fix `Preconditions.checkNotNull(TEST_RESOURCE);` in this time?

Reply:
The code is not modified at this PR.